src/main/java/com/uid2/shared/auth/ClientKey.java

```diff
@@ -110,6 +113,7 @@ public String getName() {
         return name;
     }
 
+    @Deprecated // N/A
```

Comment:
Why are these deprecated?

Reply:
I've marked it that way because it doesn't make sense to have constructors that sets a default site ID

The only times we've used those constructors is under three situations.

**Situation 1 - Never (N/A)**

**Situation 2 - Test cases only**
It would be better to create a builder that provides more meaningful defaults. Otherwise, we would have production code that are actually only used for testing, and also makes it a bit more confusing (e.g. why are we defaulting site ID to 0 or -1?).

**Situation 3 - Builder pattern where it's not really needed**
There are places in our code, specifically uid2-admin, where we use a constructor that has default values, but use the `.with()` functions to provider values for those default-initialized fields anyway.